src/Analysis/Ast/Test/ImportTests.cs

```diff
@@ -234,5 +238,57 @@ import sys
             var analysis = await GetAnalysisAsync(code);
             analysis.Should().HaveVariable("x").OfType(BuiltinTypeId.Int);
         }
+
+        [TestMethod, Priority(0)]
+        public async Task ModuleInternalImportSys() {
+            var appUri = TestData.GetTestSpecificUri("app.py");
+            await TestData.CreateTestSpecificFileAsync(Path.Combine("package", "__init__.py"), "from . import m1\nimport sys");
```

Comment:
Why do you need real files in file system?

Reply:
Will Importing/dependency resolution need something to load?